### ISSUE TITLE
[3.2] Liege of the Primordial elementalist ascendancy does not work

### ISSUE THREAD

**Issue by AIElemental** [Openarl/PathOfBuilding#995]
My scenario:
lvlv 70 Witch, Elementalist
2 skills - fireball and flame golem, both active
no items, or passives
i toggle Liege of the primordial ascendancy point and nothing happens. DPS of fireball stays the same
though LotP reads - increased damage per golem and increased golem buff per golem.
Am i missing some toggle that enables this mechanics or this is a bug?
Also can i help solve it?

**Comment by AIElemental** [comment]
added fix in https://github.com/Openarl/PathOfBuilding/pull/996
@Openarl please see when you have time

**Comment by skimbre** [comment]
I agree with you same here both Liege of the promordial and Elemancer both arent working.

**Comment by cakegod** [comment]
+1

**Comment by Dagda47** [comment]
+1

**Comment by Bhaladas** [comment]
Same. 2 Golems active (Fire & Lightning) no DPS change when enabling or disabling Liege of the Primordial.

Also Lighting Golem wrath aura makes no DPS change either.

**Comment by AIElemental** [comment]
@Bhaladas wrath aura worked for me, maybe it affects not all sources of damage?

**Comment by PathfollowerXd** [comment]
seems a little vague, isn't the [elementalist](https://mmoauctions.com/news/poe-elementalist-build-2019) supposed to be a starter class?
